_resources/contributor.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Contributors
+summary: Stay up to date with the codebase and discover notable RFCs, PRs and more
+link: https://pytorch.org/resources/contributors
```

Comment:
It is possible to change this link so that it'll work locally? Right now it sends you to the live site?

Reply:
You would have to change `link:` to `http://localhost:4000/resources/contributors` to get it to work locally. The file that is linked to the card only exists in #90 so you would get routed to an error page.